oviewer/keybind.go

```diff
@@ -365,7 +365,7 @@ func (k KeyBind) String() string {
 	k.writeKeyBind(&b, actionLastSection, "last section")
 	k.writeKeyBind(&b, actionFollowSection, "follow section mode toggle")
 	k.writeKeyBind(&b, actionSectionNum, "number of section header lines")
-	k.writeKeyBind(&b, actionHideOther, "toggle hide other section")
+	k.writeKeyBind(&b, actionHideOther, "hide other section toggle")
```

Comment:
It's unclear if you hide other section,or iftou hide a section named "other"

So maybe this


```suggestion
	k.writeKeyBind(&b, actionHideOther,`"hide "other" section toggle`)
```

Reply:
Thank you.